README.md

```diff
@@ -47,6 +47,7 @@ See public/licenses.txt
 * A Ruby manager such as [RVM](http://rvm.io/) or [rbenv](https://github.com/rbenv/rbenv) is strongly recommended.
 * (For shapefile support) access to an [ogre](http://ogre.adc4gis.com) server
 * (For placename completion) a [GeoNames](http://www.geonames.org) account
+* For automatic spatial and temporal extraction from the search text, clone and set up an [EDSC-NLP](https://github.com/mightynimble/edsc-nlp) server
```

Comment:
We have the ability to request additional repos from the NASA GitHub org.

On Aug 31, 2016 10:35 PM, "Mark Reese" <notifications@github.com> wrote:

> In README.md
> <https://github.com/nasa/earthdata-search/pull/251#discussion_r77106368>:
>
> > @@ -47,6 +47,7 @@ See public/licenses.txt
> >  * A Ruby manager such as [RVM](http://rvm.io/) or [rbenv](https://github.com/rbenv/rbenv) is strongly recommended.
> >  * (For shapefile support) access to an [ogre](http://ogre.adc4gis.com) server
> >  * (For placename completion) a [GeoNames](http://www.geonames.org) account
> > +* For automatic spatial and temporal extraction from the search text, clone and set up an [EDSC-NLP](https://github.com/mightynimble/edsc-nlp) server
>
> I'll ping the higher-ups to see what they think. My gut tells me it
> shouldn't live in a personal account too, but I'm not sure where to put it
> either. I'll take that action item.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/nasa/earthdata-search/pull/251/files/f068791d4d903c42face4463846f7777b7ce8670..e605935d160b6a127a765d39ae9de0f02cf71c57#r77106368>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AADuwMU0b2p6Z6470GlFCsjhazKJmlURks5qljnygaJpZM4JxrgI>
> .
>